modules/shared/src/test/scala/org/tessellation/schema/TransactionSuite.scala

```diff
@@ -18,13 +18,15 @@ object TransactionSuite extends ResourceSuite with Checkers {
   override type Res = KryoSerializer[IO]
 
   override def sharedResource: Resource[IO, Res] =
-    KryoSerializer.forAsync[IO](sharedKryoRegistrar, List.empty, setReferences = true)
+    KryoSerializer.forAsync[IO](sharedKryoRegistrar.view.mapValues(_.value).toMap, List.empty, setReferences = true)
```

Comment:
This mapping is repeated in the code. Please place it inside of the `forAsync` method - type of the `registrar` parameter has to be changed to `Map[Class[_], KryoRegistrationId[_]]`.